scripts/end_to_end_test/smart_contract_tests/ledger_context.py

```diff
@@ -73,10 +78,13 @@ def run(options):
     contract = Contract(CONTRACT_TEXT)
 
     # deploy the contract to the network
-    api.sync(api.contracts.create(entity1, contract, 2000))
+    status = api.sync(api.contracts.create(entity1, contract, 2000))[0]
+
+    block_number = status.exit_code
```

Comment:
Rename to `block_number_state` maybe?

Reply:
This value does not come from a state variable - it's returned directly by `@init`.